code/game/objects/items/rcd/RPD.dm

```diff
@@ -202,8 +204,10 @@ GLOBAL_LIST_INIT(transit_tube_recipes, list(
 	var/reprogram_speed = 0.2 SECONDS
 	///Category currently active (Atmos, disposal, transit)
 	var/category = ATMOS_CATEGORY
-	///Piping layer we are going to spawn the atmos device in
-	var/piping_layer = PIPING_LAYER_DEFAULT
+	///All pipe layers we are going to spawn the atmos devices in
+	var/pipe_layers = PIPE_LAYER(3)
+	///Are we only laying a single layer per click
+	var/single_layer = TRUE
```

Comment:
This feels like it shouldn't be a variable on the object and should instead be calculated on use.